internal/testrunner/runners/system/runner.go

```diff
@@ -995,7 +993,7 @@ func (r *runner) prepareScenario(config *testConfig, ctxt servicedeployer.Servic
 }
 
 func (r *runner) removeServiceStateDir() error {
-	dirPath := r.serviceStateFolder
+	dirPath := filepath.Dir(r.serviceStateFilePath)
 	err := os.RemoveAll(dirPath)
```

Comment:
Maybe we should remove the file only, in case we use the directory later for other states. Leaving the directory there should not be a problem.

Reply:
That makes totally sense.
Update this function to just delete the file, and update the tests accordingly.